macros/ano-20macros.cfg

```diff
@@ -473,6 +473,11 @@
         {MSGm_Lorin _"Be quiet, stupid man. No one wants to hear your lies."}
     [/event]
 
+    [event]
+        {KILLEDBY (Uri van Roe) (Reme Carrenemoe)}
+        {MSGm_Reme _"I always was suspicious of you, Uri, even before you betrayed us... time for you to die."}
```

Comment:
No need for "time for you to die".

Reply:
I feel like Reme should say SOMETHING else besides his first sentence, though... any suggestions on what to change it to?

Reply:
Something about him being right all along, or karma catching up to Uri.

Reply:
Do they even have a concept of karma? I dunno if that fits with the Akladian religion; maybe something about punishment from their god instead? 
(not to get too into theology, but my understanding is that "karma" is a force that operates independently from any particular god's will, whereas the Akladians don't see anything happening except by their god's will... I could be wrong tho.)

Reply:
Is 6f04548 better?

Reply:
ok should be fine as of d8d2616 now